dgl/src/Color.cpp

```diff
@@ -105,7 +107,36 @@ Color::Color(const Color& color1, const Color& color2, float u) noexcept
 
 Color Color::fromHSL(float hue, float saturation, float lightness, float alpha)
 {
+#if defined(HAVE_DGL)
     return nvgHSLA(hue, saturation, lightness, static_cast<uchar>(getFixedRange(alpha)*255.0f));
+#else
+    float m1, m2;
+    Color col;
+    hue = fmodf(hue, 1.0f);
+    if (hue < 0.0f) hue += 1.0f;
+    fixRange(saturation);
+    fixRange(lightness);
+    m2 = lightness <= 0.5f ? (lightness * (1 + saturation)) : (lightness + saturation - lightness * saturation);
+    m1 = 2 * lightness - m2;
+    auto hue_ = [](float h, float m1, float m2) -> float
```

Comment:
DPF needs c++98 compat in order to build for MacOS 10.6 and PPC.
Please do not use auto in DPF code.